### PR TITLE
Laravel set list

### PR DESCRIPTION
**Pull request by peterfox**
Added to the Laravel set list for https://github.com/rectorphp/getrector-com/pull/2556

### CODE REVIEW

File: src/RuleFilter/Bridge/LaravelSetProvider.php [hunk 1]
```diff
@@ -8,13 +8,40 @@
 use Rector\Set\Contract\SetProviderInterface;
 use Rector\Set\ValueObject\Set;
 use RectorLaravel\Set\LaravelSetList;
+use RectorLaravel\Set\Packages\Livewire\LivewireSetList;
 
 /**
- * @todo should be part of the Laravel package, move later
  * @see https://github.com/driftingly/rector-laravel/tree/main/config/sets
  */
 final class LaravelSetProvider implements SetProviderInterface
```
Comment (TomasVotruba) [inline src/RuleFilter/Bridge/LaravelSetProvider.php:16]: Awesome!

While we're at it, this `LaravelSetProvider` class should be part of https://github.com/driftingly/rector-laravel/ code.

Could you add it there, so we can just re-use the dependency?

Comment (peterfox) [reply src/RuleFilter/Bridge/LaravelSetProvider.php:16]: Yes, of course. My only question is about `GroupName::LARAVEL`, will it be fine if the value of `laravel` is just used as the class is in the getrector repository?

**Comment by TomasVotruba** [comment]
Looking into this today :eyes:

**Comment by TomasVotruba** [comment]
Rebased and merged in https://github.com/rectorphp/getrector-com/pull/2587

Thank you :+1: 

Deploy coming soon...

**Pull request closed without merging**